README.md

```diff
@@ -12,31 +12,41 @@ You may use the `application-service` component as an example for how to add you
+    - See `components/application-service/` for an example of this.
+1) Create an Argo CD `ApplicationSet` resource in `argo-cd-apps/base/(component-name).yaml`).
     - See `application-service.yaml` for a template of how this should look.
     - The `.spec.template.spec.source.path` value should point to the directory you created in previous step.
     - The `.spec.template.spec.destination.namespace` should match the target namespace you wish to deploy your resources to.
     - The `.spec.template.spec.destination.name` should correspond to the name of the workspace you wish to deploy your resources to. There are two types of destination:
       * `redhat-appstudio-workspace-{{kcp-name}}` for all AppStudio components
       * `redhat-hacbs-workspace-{{kcp-name}}` for all HACBS components
-    - The suffix of the `.spec.template.metadata.name` should correspond to your team name, but keep the `{{kcp-name}}-` prefix for proper templating: `{{kcp-name}}-(team-name)`.
-4) Add a reference to your new `(team-name).yaml` file, to `argo-cd-apps/base/kustomization.yaml` (the reference to your YAML file should be in the `resources:` list field).
-5) Run `kustomize build (repo root)/argo-cd-apps/overlays/staging` and ensure it passes, and outputs your new Argo CD Application CR.
-6) Add an entry in `argo-cd-apps/overlays/development/repo-overlay.yaml` for your new component so you can use the preview mode for testing.
-7) Add an APIBinding (including the accepted permission claims):
+    - The suffix of the `.spec.template.metadata.name` should correspond to your team name, but keep the `{{kcp-name}}-` prefix for proper templating: `{{kcp-name}}-(component-name)`.
+1) Add a reference to your new `(component-name).yaml` file, to `argo-cd-apps/base/kustomization.yaml` (the reference to your YAML file should be in the `resources:` list field).
+1) Run `kustomize build (repo root)/argo-cd-apps/overlays/staging` and ensure it passes, and outputs your new Argo CD Application CR.
+1) Add an entry in `argo-cd-apps/overlays/development/repo-overlay.yaml` for your new component so you can use the preview mode for testing.
+1) Add an APIBinding:
    - For an AppStudio component to [apibindings/appstudio](apibindings/appstudio)
    - For an HACBS component to [apibindings/hacbs](apibindings/hacbs)
-8) Open a PR for all the above.
+1) Generate APIExports for overlays if component APIExports requires identityHashes:
+   - Execute `hack/generate-apiexport-overlays.sh components/(component-name)`
+   - The script generates `apiexport.yaml` files in `overlays`
+   - `overlays/dev/apiexport.yaml` contains APIExports from the component repository
+   - `overlays/kcp-stable/apiexport.yaml` replaces `identityHash: placeholder` by value in file `identity/kcp-stable/placeholder`
+   - `overlays/kcp-unstable/apiexport.yaml` replaces `identityHash: placeholder` by value in file `identity/kcp-stable/placeholder`
```

Comment:
typo - should say by value in file with path to the unstable placeholder instead of stable

Reply:
updated, thanks